coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/functions/VersionedSqlUserDefinedFunction.java

```diff
@@ -46,13 +46,6 @@ private VersionedSqlUserDefinedFunction(SqlIdentifier opName, SqlReturnTypeInfer
     this.viewDependentFunctionName = viewDependentFunctionName;
   }
 
-  public VersionedSqlUserDefinedFunction(String name, SqlReturnTypeInference returnTypeInference,
```

Comment:
this is a public method. Do we need to preserve this signature though ?

Reply:
Added back